### DISCUSSION TITLE
Set up the person who send feedback to Langsmith

### DISCUSSION THREAD

**Discussion by HoangNguyen689** [langchain-ai/langsmith-sdk#728] [Q&A]
In the Langsmith console, Feedback tab, I see the `Created by` field.
After digging into the client code, I can't find where we can set up the feedback poster.
How can I do it?

**Comment by HoangNguyen689** [comment]
@hinthornw Would like to hear from you.

**Comment by hinthornw** [comment]
"Created By" represents a LangSmith use today. It's populated when you annotate things in the app (in the annotation queue, for instance) and isn't supported via API feedback right now - it seems like you'd like to correlate feedback with users of your application?

**Reply by HoangNguyen689** [reply]
@hinthornw 
> it seems like you'd like to correlate feedback with users of your application?

Yes, exactly! 
I have feedback from the sender and want to display it on Langsmith.
So `Created by` has a meaning other than that of the one who sends the feedback?
